### PR TITLE
expose django-q timeout and retry to env vars

### PR DESCRIPTION
**Pull request by marxide**
Allows the Django-Q config variables `timeout` and `retry` to be configured with env vars. Defaults to existing values if not configured in `.env`.
[skip ci]

### CODE REVIEW

File: webinterface/.env.template [hunk 1]
```diff
@@ -32,3 +32,6 @@ HOME_DATA_DIR=vast-pipeline-extra-data
 # PIPELINE_MAINTAINANCE_MESSAGE=Uncomment and fill to show
 MAX_PIPELINE_RUNS=3
 MAX_PIPERUN_IMAGES=200
+
+# Q_CLUSTER_TIMEOUT=172800
+# Q_CLUSTER_RETRY=172920
```
Comment (ajstewart) [inline webinterface/.env.template:37]: Should these match the default values?

Comment (marxide) [reply webinterface/.env.template:37]: The defaults are defined in `settings.py`, but I guess so for consistency

**Review by ajstewart — COMMENTED** [review]
Looking at it, my only comment would be whether the default should be set even higher than 24 hours? Just to avoid any unexpected situations in the future if someone else were to use it/set it up somewhere, because as it stands, a big job could take a long time initially.

**Comment by marxide** [comment]
Our largest runs usually take about 12 hours. I'm not sure what caused the one that exceeded the 24h timeout, but a likely scenario is that the system was under load with other processes (data transfers, Jupyter servers, or something else). The run after the one that timed out took ~12 hours.

**Comment by marxide** [comment]
We're also going to remove the measurement pairs from the database soon (WIP) which that will significantly reduce the running time.

**Review by ajstewart — APPROVED** [review]

**Pull request merged**